tmt/base.py

```diff
@@ -3735,12 +3735,11 @@ def show(self) -> None:
         """ Display the current status """
         # Prepare absolute workdir path if --id was used
         # FIXME: cast() - typeless "dispatcher" method
```

Comment:
I believe the "FIXME" comment could be deleted as it refers to the `id_ = cast(str, self.opt('id'))` line, which is no longer present.